ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.22.6-bullseye
+    tag: 1.22.7-bullseye
```

Comment:
Should be 1.23.1 now

Reply:
Went with the 1.22.7 because of the ci-lint.

Error: can't load config: the Go language version (go1.22) used to build golangci-lint is lower than the targeted Go version (1.23.1)
Failed executing command with error: can't load config: the Go language version (go1.22) used to build golangci-lint is lower than the targeted Go version (1.23.1)

Reply:
https://github.com/ONSdigital/dp-frontend-router/blob/develop/ci/lint.yml#L9  is the line that needs to be updated to remedy this (and check other files in that directory, too)